ruby_on_rails/forms_and_authentication/project_forms.md

```diff
@@ -98,6 +98,3 @@ Now we'll start morphing our form into a full Rails form using the `#form_tag` a
 5. Save this project to Git and upload to GitHub.
 
 </div>
-
```

Comment:
Per the style guide, we want to leave this section in even if it's empty - it makes it easier to add links to in the future should the need arise.

>If the lesson doesn't include any additional resources, leave this section as-is. Otherwise, replace the default bulleted resource item with your own resource, then add any additional bulleted resource items. The lesson should ideally have no more than 3-5 additional resources.

Reply:
Yes @ChargrilledChook . As this is a project, I referred to the project layout guide linked below. That doesn't include the additional resources section. So I removed it. Please let me know your comments so that we can make the changes accordingly. 

Link: https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#project-layout

PS: Sorry. I accidentally deleted the previous comment.

Reply:
You're absolutely right, I missed that it was a project. ✅